src/main/java/bossmonster/domain/Boss.java

```diff
@@ -0,0 +1,38 @@
+package bossmonster.domain;
+
+import bossmonster.util.ErrorChecker;
+
+import static bossmonster.util.Constants.*;
+
+public class Boss {
+
+    private int hp;
+    private final int initialHp;
```

Comment:
순서를 아래와 같이 해주시는게 좋을거 같아요!
(개인마다, 팀마다 다를 수 있습니다!)
```suggestion
    private final int initialHp;
    private int hp;
```